### PR TITLE
GCLOUD_PROJECT instead of GCLOUD_PROJECT_NUM

### LINKED ISSUE #90: Use GCLOUD_PROJECT instead of GCLOUD_PROJECT_NUM

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #90

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: README.md [hunk 1]
```diff
@@ -21,33 +21,11 @@ This module provides Cloud Debug support for Node.js applications. [Google Cloud
...
 
-# Change the start command of your application in package.json
-   ...
-   "scripts" {
-     "start": "node --require @google/cloud-debug server.js"
-   ...
+# Require the agent at the top of your main script.
+require('@google/cloud-debug');
 ```
 Deploy your application, and navigate to the [Debug tab][debug-tab] within the [Google Developers Console][dev-console] to set breakpoints and start debugging.
```
Comment (ofrobots) [inline README.md]: The url for dev-console needs to be updated to https://console.cloud.google.com

**Comment by ofrobots** [comment]
While you are in README.me, you can drop the warning about Node.js 5.2.0 not being supported.

File: README.md (hunk 1 above)
Comment (matthewloring) [inline README.md]: Done.

**Comment by matthewloring** [comment]
Done.

File: README.md [hunk 2]
```diff
@@ -72,9 +50,9 @@ Container Engine nodes need to also be created with the `cloud-platform` scope,
 
 If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you can still use Cloud Debug.
 
-1. You will need to specify the numeric project ID of your project. Your numeric project ID, or project number, is visible the Project information section on your project's [dashboard][dashboard].
+1. You will need to specify your project name. Your project name is visible in the [Google Cloud Console Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `particular-future-12345`. If your application is [running on Google Cloud Platform](running-on-google-cloud-platform), you don't need to specify the project name.
```
Comment (ofrobots) [inline README.md]: Put url in the section at the bottom, and user a shorter syntax.

Comment (ofrobots) [inline README.md]: *use a

Comment (matthewloring) [inline README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -110,10 +88,13 @@ As soon as that line of code is reached in any of the running instances of your
 
 
 [cloud-debugger]: https://cloud.google.com/tools/cloud-debugger/
-[dev-console]: https://console.developers.google.com/
+[dev-console]: https://console.cloud.google.com/
 [dashboard]: https://console.developers.google.com/dashboard
```
Comment (ofrobots) [inline README.md]: [dashboard] is no longer referenced (and 404's).

File: lib/debugletapi.js [hunk 4]
```diff
@@ -84,14 +84,11 @@ DebugletApi.prototype.init = function(uid, logger, callback) {
     });
   }
 
-  if (process.env.GCLOUD_PROJECT_NUM) {
-    var projectNumber = process.env.GCLOUD_PROJECT_NUM;
-    var err = /^\d+$/.test(projectNumber) ?
-      null :
-      new Error('GCLOUD_PROJECT_NUM should be numeric');
+  if (process.env.GCLOUD_PROJECT) {
+    var projectNumber = process.env.GCLOUD_PROJECT;
```
Comment (ofrobots) [inline lib/debugletapi.js]: rename variables related to this to `project` and `project_` and update the jsdoc for `this.project_`.

**Comment by ofrobots** [comment]
LGTM w/ comments.

File: README.md (hunk 3 above)
Comment (matthewloring) [inline README.md]: Done.

File: lib/debugletapi.js (hunk 4 above)
Comment (matthewloring) [inline lib/debugletapi.js]: Fixed.

**Pull request merged**